core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.spec.ts

```diff
@@ -146,11 +147,12 @@ describe('Translation Modal Component', () => {
       spyOn(translateTextService, 'init').and.callFake(
         (expId, languageCode, successCallback) => successCallback());
       component.ngOnInit();
-      expect(contextService.getEntityType()).toBe(
+      expect(OppiaAngularRootComponent.contextService.getEntityType()).toBe(
```

Comment:
Minor nit: If you are assigning Context service to OppiaAngularRootComponent on line 75, then these changes (changing `contextService` to `OppiaAngularRootComponent.contextService`) are not required. We definitely want to keep a minimum footprint of changes here.